Automattic/wp-calypso
# Happy Chat: Allow customers to receive message updates

#### Changes proposed in this Pull Request

Shows updates in Happy Chat when an operator edits a message (including an `(edited)` flag). This is a high-value change for Happiness Engineers, especially in correcting typos that have the wrong link or send the wrong message.

https://user-images.githubusercontent.com/518059/124288331-ab061c80-db16-11eb-8177-fed724917d28.mp4

#### Testing instructions

Open a chat, send and edit a few messages from the Happy Chat HUD (including messages with links). Ensure that the customer experience looks and feels as expected — all edited messages are marked as such and edited links direct you to the correct URL.
